hibernate-community-dialects/src/main/java/org/hibernate/community/dialect/TimesTenSqlAstTranslator.java

```diff
@@ -143,4 +145,56 @@ protected boolean supportsRowValueConstructorSyntaxInInList() {
 	protected boolean supportsRowValueConstructorSyntaxInQuantifiedPredicates() {
 		return false;
 	}
+
+	@Override
```

Comment:
Since this function is only used in TimesTen, could you please remove it from `AbstractSqlAstTranslator`?